ui/src/modules/Settings/Credentials/Sections/MetricProvider/styled.ts

```diff
@@ -78,14 +78,35 @@ const TestConnectionButton = styled(Button.Default)`
 const HealthSwitch = styled(Switch)`
   justify-content: start;
 
+  i {
+    width: 35px;
+    height: 20px;
+  }
+
+  i::after {
+    width: 14px;
+    height: 14px;
+  }
+
+  i::before {
+    width: 31px;
+    height: 16px;
+  }
```

Comment:
What do you think put this together?

```suggestion
  i {
    width: 35px;
    height: 20px;
    
    ::after {
      width: 14px;
      height: 14px;
    }
    
    ::before {
      width: 31px;
      height: 16px;
    }
  }
```